php/commands/help.php

```diff
@@ -90,11 +90,8 @@ private static function indent( $whitespace, $text ) {
 	}
 
 	private static function pass_through_pager( $out ) {
-		if ( Utils\is_windows() ) {
-			// no paging for Windows cmd.exe; sorry
-			echo $out;
-			return 0;
-		}
+
+		$pager = Utils\is_windows() ? 'more' : 'less -r';
```

Comment:
I believe so, it was even a DOS command. This is what I found: [more command availability](http://www.computerhope.com/morehlp.htm#availability).